### PR TITLE
Added seasonal events

### PR DESCRIPTION
**Pull request by SavageSpaceLion**
@vpierson please review

### CODE REVIEW

File: SavageSpaceLion-Blaine.md [hunk 1]
```diff
@@ -8,7 +8,13 @@
 ### Places the locals like
 - Lockness park
 ![Lochness Monster](http://nerdist.com/wp-content/uploads/2016/04/private-life-sherlock-holmes-loch-ness.png)
+- Curling club
```
Comment (vpierson) [inline SavageSpaceLion-Blaine.md]: :scream: Can't believe you forgot this!

File: SavageSpaceLion-Blaine.md [hunk 2]
```diff
@@ -8,7 +8,13 @@
...
 
 ### Places tourists like
 - Northtown Mall
 - Brunswick Bowl
+
+### Seasonal Events
+- USA Cup (Summer)
+- Garage sale weekend (Fall)
+- Food truck fair (Fall)
```
Comment (vpierson) [inline SavageSpaceLion-Blaine.md]: :fallen_leaf:

**Review by vpierson — CHANGES_REQUESTED** [review]
More emojis please! :smile_cat:

**Comment by SavageSpaceLion** [comment]
How rude!

**Review by vpierson — APPROVED** [review]

**Pull request merged**